### PR TITLE
Added Simmer Index and Simmer Zone sensors

### PR DESCRIPTION
**Pull request by ahmadtawakol**
This is based on another fork of thermal_comfort by @lymanepp at https://github.com/lymanepp/thermal_comfort. I adapted the code to work with this plugin. The only thing missing is the translations for the Simmer Zone strings.

### CODE REVIEW

**Comment by lymanepp** [comment]
Please give credit where it is due.

File: README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ Thermal Comfort provides the following calculated sensors for Home Assistant:
  * Thermal Perception `perception`
  * Frost point `frostpoint`
  * Frost Risk `frostrisk`
+ * Simmer Index `simmerindex`
+ * Simmer Zone `simmerzone`
```
Comment (rautesamtr) [inline README.md:13]: Could you also add them to the list of available sensors for sensor_types?

File: custom_components/thermal_comfort/sensor.py [hunk 2]
```diff
@@ -76,6 +79,16 @@
...
 
+SIMMER_COOL= 'Cool'
+SIMMER_SLIGHTLY_COOL = 'Slightly cool'
+SIMMER_COMFORTABLE = 'Comfortable'
+SIMMER_SLIGHTLY_WARM = 'Slightly warm'
+SIMMER_INCREASING_DISCOMFORT = 'Increasing discomfort'
+SIMMER_EXTREMELY_WARM = 'Extremely warm'
+SIMMER_DANGER_OF_HEATSTROKE = 'Danger of heatstroke'
+SIMMER_EXTREME_DANGER_OF_HEATSTROKE = 'Extreme danger of heatstroke'
+SIMMER_CIRCULATORY_COLLAPSE_IMMINENT = 'Circulatory collapse imminent'
```
Comment (rautesamtr) [inline custom_components/thermal_comfort/sensor.py:90]: Could you change those to short `snake_case` strings and add the proper string to sensor.en.json similar to the other translated strings?

**Review by rautesamtr — CHANGES_REQUESTED** [review]
I'd suggest you squash the commits and add @lymanepp as Co-authored-by to give credit. Would that be ok for you @lymanepp?

**Comment by lymanepp** [comment]
> I'd suggest you squash the commits and add @lymanepp as Co-authored-by to give credit. Would that be ok for you @lymanepp?

Yes, that sounds good to me.

**Comment by rautesamtr** [comment]
@ahmadtawakol If your up for it, it would also be nice if you would add some test cases. Otherwise i can do it myself.

**Comment by lymanepp** [comment]
> > I'd suggest you squash the commits and add @lymanepp as Co-authored-by to give credit. Would that be ok for you @lymanepp?
> 
> Yes, that sounds good to me.

I don't see that any credit was given.

**Comment by ahmadtawakol** [comment]
@lymanepp I closed this PR and created a new one. I added you as a co-author to the commit.

**Comment by lymanepp** [comment]
> @lymanepp I closed this PR and created a new one. I added you as a co-author to the commit.

Thanks! 👍

**Pull request closed without merging**